test/test_manifest.json

```diff
@@ -1015,6 +1015,12 @@
        "firstPage": 2,
        "type": "eq"
     },
+    {  "id": "doc_actions",
```

Comment:
There's already a file with an *almost* identical name, see `docactions.js`.
Can we perhaps use a that file for these new tests as well, or at least use a bit less similar filenames please?